ui/raidboss/emulator/data/network_log_converter/ParseLine.ts

```diff
@@ -23,13 +23,14 @@ import { LineEvent36 } from './LineEvent0x24';
 import { LineEvent37 } from './LineEvent0x25';
 import { LineEvent38 } from './LineEvent0x26';
 import { LineEvent39 } from './LineEvent0x27';
+import LogRepository from './LogRepository';
 
 export default class ParseLine {
-  static parse(repo, line) {
+  static parse(repo: LogRepository, line: string): LineEvent | false {
```

Comment:
As in the other patch, what about `LineEvent | undefined`?

Reply:
Adjusted for next commit.